components/esm/cellref.cpp

```diff
@@ -19,10 +40,8 @@ void ESM::CellRef::loadId(ESMReader &esm, bool wideRefNum)
     if (esm.isNextSub ("NAM0"))
         esm.skipHSub();
 
-    if (wideRefNum)
-        esm.getHNT (mRefNum, "FRMR", 8);
-    else
-        esm.getHNT (mRefNum.mIndex, "FRMR");
+    if (!ignoreRefNum)
```

Comment:
Might be best for Zini to comment.  I basically fixed a few bugs without thinking too much about what was already present.